base/c.jl

```diff
@@ -21,9 +21,9 @@ if is_windows()
     typealias Culong UInt32
     typealias Cwchar_t UInt16
 else
-    typealias Clong Int
```

Comment:
I changed from aliases to normal names, so it is clearer what's the difference between Windows and the rest of systems.